jbpm-services/jbpm-kie-services/src/main/java/org/jbpm/kie/services/impl/ProcessServiceImpl.java

```diff
@@ -155,14 +155,21 @@ public Long startProcess(String deploymentId, String processId, CorrelationKey c
 	}
 
 	@Override
-	public void abortProcessInstance(Long processInstanceId) {
-		ProcessInstanceDesc piDesc = dataService.getProcessInstanceById(processInstanceId);
-		if (piDesc == null || piDesc.getState() != 1) {
-			throw new ProcessInstanceNotFoundException("Process instance with id " + processInstanceId + " was not found");
-		}
-		DeployedUnit deployedUnit = deploymentService.getDeployedUnit(piDesc.getDeploymentId());
+    public void abortProcessInstance(Long processInstanceId) {
+        ProcessInstanceDesc piDesc = dataService.getProcessInstanceById(processInstanceId);
+        if (piDesc == null || piDesc.getState() != 1) {
```

Comment:
Could change to ProcessInstance.STATE_ACTIVE ?

Reply:
done, all instances using 1 was changed to use ProcessInstance.STATE_ACTIVE